### PR TITLE
Changes made for masters Utrecht and Logo

### PR DESCRIPTION
**Pull request by mndore**
- Made a change in the positioning of the logo. 
- Added 3 uu masters
- Changed nav bar

### CODE REVIEW

File: nsvki/views/page.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ def index():
 def amsterdam():
     return render_template('en/amsterdam.htm', title="Masters in Amsterdam")
```
Comment (LorianColtof) [inline nsvki/views/page.py]: PEP8 requires that there are two blank lines between functions.

File: nsvki/views/page.py [hunk 2]
```diff
@@ -21,6 +21,9 @@ def index():
...
 
+@blueprint.route('/masters/utrecht', methods=['GET'])
+def utrecht():
+    return render_template('en/utrecht.htm', title="Masters in Utrecht")
```
Comment (LorianColtof) [inline nsvki/views/page.py]: Same here

File: nsvki/templates/en/utrecht.htm [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            <li><a href="#gt">Game Media and Technology</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class='row'>
+        <h2><a name="ai"></a>Artificial Intelligence</h2>
+        <h4><i>Utrecht University</i></h4>
+        <div class='col-md-2'><img
+          class='img-responsive center-block logo'
+          src="{{ static_url('/static/img/logo_uu.png') }}"
```
Comment (LorianColtof) [inline nsvki/templates/en/utrecht.htm]: The UU logo is missing.

File: nsvki/views/page.py (hunk 1 above)
Comment (mndore) [reply nsvki/views/page.py]: Changed page.py to 2 blank lines both cases

File: nsvki/templates/en/utrecht.htm (hunk 3 above)
Comment (mndore) [reply nsvki/templates/en/utrecht.htm]: I added the logo file

**Pull request merged**